.github/workflows/pythonpublish.yml

```diff
@@ -0,0 +1,45 @@
+name: Upload Python Package
```

Comment:
we need to take a look at how to do this, probably discuss how we want to publish the three environments (cpu, gpu and spark)